src/ImageSharp/Processing/Quantization/IQuantizer.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+using SixLabors.ImageSharp.PixelFormats;
+using SixLabors.ImageSharp.Processing.Dithering.ErrorDiffusion;
+using SixLabors.ImageSharp.Processing.Quantization.FrameQuantizers;
+
+namespace SixLabors.ImageSharp.Processing.Quantization
+{
+    /// <summary>
+    /// Provides methods for allowing quantization of images pixels with configurable dithering.
+    /// </summary>
+    public interface IQuantizer
+    {
+        /// <summary>
+        /// Gets a value indicating whether to apply dithering to the output image.
+        /// </summary>
+        bool Dither { get; }
```

Comment:
As with below `DitherType`  property not needed on interface and not even needed on classes as it can be infered by weather the `IErrorDiffuser` is set or not as they are immutable classes.